target/linux/ath79/dts/ar9344_tplink_tl-wdr3600-v1.dts

```diff
@@ -4,6 +4,6 @@
 #include "ar9344_tplink_tl-wdr4300.dtsi"
 
 / {
-	model = "TP-Link WDR3600";
-	compatible = "tplink,tl-wdr3600", "qca,ar9344";
+	model = "TP-Link WDR3600 v1";
```

Comment:
While you are at it:
`TP-Link WDR3600 v1` --> `TP-Link TL-WDR3600 v1` for consistency 😄